### PR TITLE
Support bitmap urls in large-icon

### PR DESCRIPTION
**Pull request by baldurpan**
Needed a way to show profile pictures in Android notifications. This was my solution.

### CODE REVIEW

File: android/src/main/java/com/evollu/react/fcm/FIRLocalMessagingHelper.java [hunk 1]
```diff
@@ -110,11 +114,17 @@ public void sendNotification(Bundle bundle) {
             //large icon
             String largeIcon = bundle.getString("large-icon");
             if(largeIcon != null && android.os.Build.VERSION.SDK_INT >= android.os.Build.VERSION_CODES.LOLLIPOP){
-                int largeIconResId = res.getIdentifier(largeIcon, "mipmap", packageName);
-                Bitmap largeIconBitmap = BitmapFactory.decodeResource(res, largeIconResId);
+                if (largeIcon.startsWith("http://") || largeIcon.startsWith("https://")) {
+                    Bitmap bitmap = getBitmapFromURL(largeIcon);
+                    notification.setLargeIcon(bitmap);
+                    //notification.setStyle(new Notification.BigPictureStyle().bigPicture(bitmap));
```
Comment (evollu) [inline android/src/main/java/com/evollu/react/fcm/FIRLocalMessagingHelper.java]: can you remove the comment if not needed?

**Comment by evollu** [comment]
thanks

**Comment by ptomasroos** [comment]
A small notice: I think it would be really sweet to use fresco instead of just opening another HTTP Connection here. Just a small improvement. That would allow images to be cached and use already retrieved image from cache ect. Many reasons to

**Comment by evollu** [comment]
@ptomasroos a pull request will be great. I don't really know android

**Comment by baldurpan** [comment]
I'm also not much of a Java developer. It would have been super nice to be able to push icon URLs without using local notifications through custom_notification but I'm not quite sure how to pull it off, or if it's even possible.

**Pull request merged**